cpp/common/time.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) ZeroC, Inc.
+
+#include <chrono>
+#include <iostream>
+#include <string>
+
+namespace common
+{
+    /// Format a time point as a string.
+    /// @return The formatted time point.
+    std::string formatTime(std::chrono::system_clock::time_point timePoint)
+    {
+        std::time_t time = std::chrono::system_clock::to_time_t(timePoint);
+        std::tm timeInfo;
+
+#ifdef _MSC_VER
+        if (localtime_s(&timeInfo, &time))
+        {
+            std::cerr << "localtime_s error" << std::endl;
```

Comment:
We can throw `runtime_error` it would be very rare that this fails but still seems clear.

Reply:
I'd prefer to just print then return an empty string. Otherwise we'd need to worry a little about exception handling.

Reply:
I agree, I am not fond of reporting errors via cerr.